### ISSUE TITLE
In xcode 6 beta4 building shows a couple of warnings

### ISSUE THREAD

**Issue by philosopherdog** [davedelong/CHCSVParser#67]
1) CHCSVParser.m:862:1: Designated initializer missing a 'super' call to a designated initializer of the super class
2) CHCSVParser.m:863:18: Designated initializer should only invoke a designated initializer on 'super'

**Comment by philosopherdog** [comment]
I'm just looking this problem over again. The issue is that - (instancetype)initWithObjects:(const id [])objects forKeys:(const id<NSCopying> [])keys count:(NSUInteger)cnt is the designated initializer in the superclass and this subclass overrides this method and calls an overridden convenience initializer from this overridden designated initializer. So, we've switched the designated initializer from that found in the superclass to the subclass. I'm not entirely sure how to solve this, but one workaround is to override this method with an NSAssert to prevent it from being called directly from CHCSVParser. You could just comment out this overridden designated initializer but then I suppose it's possible that somebody could call this method from CHCSVParser and generate an unexpected result.

**Comment by edwardmp** [comment]
I am no longer seeing warnings in Xcode 6, so I guess this issue can be closed @davedelong.

**Comment by philosopherdog** [comment]
I'm still seeing it in the release version of Xcode 6.

**Comment by edwardmp** [comment]
You are right; my dependency still used my own fixed branch. It seems @davedelong has implemented a fix, however CocoaPods.spec is still set to the tag 2.1.0 of July 2014 so any commits after that are not included. If @davedelong would release a new tag, the warnings would disappear.

**Comment by edwardmp** [comment]
As a workaround you could temporarily use this in your Podfile:
pod 'CHCSVParser', :git => 'https://github.com/davedelong/CHCSVParser.git'

This just uses the master branch instead of tags

**Comment by philosopherdog** [comment]
Ok, cool. @edwardmp thanx for the info.
